// Copyright 2021 The ChromiumOS Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto2";

package dlp;

option optimize_for = LITE_RUNTIME;

import "dlp_service.proto";

message DlpFuzzer {
  required SetDlpFilesPolicyRequest set_dlp_files_policy_request = 1;
  required AddFileRequest add_file_request = 2;
  required RequestFileAccessRequest request_file_access_request = 3;
  required GetFilesSourcesRequest get_files_sources_request = 4;
  required CheckFilesTransferRequest check_files_transfer_request = 5;
}